hack/generate-manifest.sh

```diff
@@ -250,7 +250,7 @@ fi
 TMP_DIR=$(mktemp -d $THIS_DIR/../build/yamls/chart-values.XXXXXXXX)
 
 if $IPSEC; then
-    HELM_VALUES+=("trafficEncryptionMode=ipsec" "tunnelType=gre")
+    HELM_VALUES+=("multicast.enabled=false" "trafficEncryptionMode=ipsec" "tunnelType=gre")
```

Comment:
same comment as above, I don't think we need this

Reply:
removed